gcsio/src/test/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageFileSystemNewIntegrationTest.java

```diff
@@ -101,11 +120,62 @@ public void mkdirs_shouldCreateNewDirectory() throws Exception {
     String dirObject = getTestResource();
     URI dirObjectUri = new URI("gs://" + bucketName).resolve(dirObject);
 
+    gcsfsIHelper.mkdir(
+        bucketName, dirObject); // create directory before hand without tracking requests
```

Comment:
Move comment on before the `mkdir` call and re-format code.

Reply:
Done